docs/src/modules/ROOT/pages/introduction/introduction.adoc

```diff
@@ -1,287 +0,0 @@
-
-Soft constraints help optimize plans based on the business goals, for instance, minimize travel time between customer visits or assign employees to their preferred shifts.
-
-To help determine the quality of the solution, plans are assigned a score with values for hard, medium, and soft constraints.
-
-`"0hard/-257medium/-6119520soft"`
-
-From this example score we can see zero hard constraints were broken, while both the medium and soft scores have negative values (the scores do not show how many constraints were broken, but values associated with those constraints).
-
-Because breaking hard constraints would result in an infeasible solution,
-a solution that breaks zero hard constraints and has a soft constraint score of -1,000,000 is better
-than a solution that breaks one hard constraint and has a soft constraint score of 0.
-
-The weight of constraints can be tweaked to adjust their impact on the solution.
-
-[#timefoldSolverStatus]
-== Status of Timefold Solver
-
-Timefold Solver is 100% pure Java^TM^ and runs on Java {java-version} or higher.
-It xref:integration/integration.adoc#integration[integrates very easily] with other Java^TM^, Python and other technologies.
-Timefold Solver works on any Java Virtual Machine and is compatible with the major JVM languages and all major platforms.
-It also supports Kotlin and Python.
-
-image::introduction/compatibility.png[align="center"]
-
-Timefold Solver is stable, reliable and scalable.
-It has been heavily tested with unit, integration, and stress tests, and is used in production throughout the world.
-One example handles over 50 000 variables with 5000 values each, multiple constraint types and billions of possible constraint matches.
-
-We offer two editions of Timefold Solver.
-
-[#communityEdition]
-=== Timefold Solver Community Edition
-
-Timefold Solver Community Edition is _open source_ software,
-released under http://www.apache.org/licenses/LICENSE-2.0.html[the Apache License 2.0].
-This license is very liberal and allows reuse for commercial purposes.
-Read http://www.apache.org/foundation/licence-FAQ.html#WhatDoesItMEAN[the layman's explanation].
-
-Timefold Solver Community Edition is available in <<useWithMavenGradleEtc,the Maven Central Repository>>.
-It is and will always be free.
-The overwhelming majority of solver features will always be available in the Community Edition.
-Most users will be able to solve their planning problems with the Community Edition.
-
-[#enterpriseEdition]
-=== Timefold Solver Enterprise Edition
-
-Timefold Solver Enterprise Edition is a commercial product
-that offers xref:enterprise-edition/enterprise-edition.adoc#enterpriseEditionFeatures[additional features]
-to scale out to very large datasets.
-To find out more, see xref:enterprise-edition/enterprise-edition.adoc[Enterprise Edition section] of this documentation.
-
-[#backwardsCompatibility]
-== Backwards compatibility
```

Comment:
It this gone now?

Reply:
As described in the PR: "Note on Backwards Compatibility. I think this does not belong in an Intro. I have removed it for now as I think the expectation is always that this information should flow through the Changelog."

You ok with that or should I bring it back somewhere else?

Reply:
When "restructuring" the chapter, I didn't expect things to disappear. While some parts I won't miss, this is IMO an important piece of content. But I agree with you that it's not an introduction material.

Reply:
It makes sense to me to add it under the "Upgrading Timefold Solver" heading. I'll make a change to re-add it there.

Reply:
Re-added under Upgrading Timefold Solver